libkbfs/bserver_remote_test.go

```diff
@@ -88,18 +102,50 @@ func (fc *FakeBServerClient) GetBlock(ctx context.Context, arg keybase1.GetBlock
 		return keybase1.GetBlockRes{}, err
 	}
 
-	fc.blocksLock.Lock()
-	defer fc.blocksLock.Unlock()
-	getRes, ok := fc.blocks[arg]
-	if !ok {
-		return keybase1.GetBlockRes{}, fmt.Errorf("No such block: %v", arg)
+	id, err := BlockIDFromString(arg.Bid.BlockHash)
+	if err != nil {
+		return keybase1.GetBlockRes{}, err
 	}
-	return getRes, nil
+
+	tlfID, err := ParseTlfID(arg.Folder)
+	if err != nil {
+		return keybase1.GetBlockRes{}, err
+	}
+
+	// Always use this block context since the RPC API drops the
+	// real context.
```

Comment:
Clarified comment. I meant that the Get RPC ignores the ref nonce and the writer argument.